src/js/containers/covid19/awardSpendingAgency/AwardSpendingAgencyTableContainer.jsx

```diff
@@ -0,0 +1,242 @@
+/**
+ * AwardSpendingAgencyTableContainer.jsx
+ * Created by James Lee 6/24/20
+ */
+
+import React, { useState, useEffect, useCallback } from 'react';
+import { useSelector } from 'react-redux';
+import ResultsTableLoadingMessage from 'components/search/table/ResultsTableLoadingMessage';
+import ResultsTableErrorMessage from 'components/search/table/ResultsTableErrorMessage';
+import PropTypes from 'prop-types';
+import { Table, Pagination } from 'data-transparency-ui';
+import { awardTypeGroups } from 'dataMapping/search/awardType';
+import CSSTransitionGroup from 'react-transition-group/CSSTransitionGroup';
+import { awardSpendingAgencyTableColumns, awardSpendingAgencyTableColumnFieldMapping, awardSpendingAgencyTableTabs } from 'dataMapping/covid19/awardSpendingAgency/awardSpendingAgencyTableTabs';
+import { fetchAwardSpendingByAgency, fetchFaceValueOfLoansByAgency } from 'helpers/disasterHelper';
+import BaseAwardSpendingByAgencyRow from 'models/covid19/awardSpendingAgency/BaseAwardSpendingByAgencyRow';
+
+
+const propTypes = {
+    type: PropTypes.string.isRequired
+};
+
+const AwardSpendingAgencyTableContainer = (props) => {
+    const [currentPage, changeCurrentPage] = useState(1);
+    const [pageSize, changePageSize] = useState(10);
+    const [totalItems, setTotalItems] = useState(0);
+    const [sort, setSort] = useState('obligation');
+    const [order, setOrder] = useState('desc');
+    const updateSort = (field, direction) => {
+        setSort(field);
+        setOrder(direction);
+    };
+    const [results, setResults] = useState([]);
+    const [loading, setLoading] = useState(true);
+    const [error, setError] = useState(false);
+    const defCodes = useSelector((state) => state.covid19.defCodes);
+
+    const parseAwardSpendingByAgency = (data) => {
+        const parsedData = data.map((item) => {
+            const awardSpendingByAgencyRow = Object.create(BaseAwardSpendingByAgencyRow);
+            awardSpendingByAgencyRow.populate(item);
+
+            let rowChildren = [];
+            if (item.children && item.children.length > 0) {
+                rowChildren = item.children.map((childItem) => {
+                    const awardSpendingByAgencyChildRow = Object.create(BaseAwardSpendingByAgencyRow);
+                    awardSpendingByAgencyChildRow.populate(childItem);
+                    return awardSpendingByAgencyChildRow;
+                });
+            }
+
+            if (rowChildren && rowChildren.length > 0) {
+                Object.defineProperty(awardSpendingByAgencyRow, "children", {
+                    value: rowChildren
+                });
+            }
+
+
+            let link = awardSpendingByAgencyRow.name;
+            const code = awardSpendingByAgencyRow.code;
```

Comment:
This should use the `id` property for now (internal/ database id). When we launch agency profile v2, the new url format will use `code`.